d3ploy/demand_driven_deployment_inst.py

```diff
@@ -297,15 +330,18 @@ def calc_diff(self, commod, time):
             self.commodity_supply[commod][time] = 0.0
         supply = self.predict_supply(commod)
 
-        if self.buffer_type_dict[commod] == 'perc':
-            demand = self.predict_demand(
-                commod, time) * (1 + self.buffer_dict[commod])
-        elif self.buffer_type_dict[commod] == 'float':
-            demand = self.predict_demand(
-                commod, time) + self.buffer_dict[commod]
+        if self.buffer_type_dict[commod] == "perc":
+            demand = self.predict_demand(commod, time) * (
+                1 + self.buffer_dict[commod]
+            )
+        elif self.buffer_type_dict[commod] == "float":
```

Comment:
This seems like it should be "relative" vs. "absolute" or "percentage" vs "constant". Float is a data type and since percents can be floats, putting these at odds with one another feels fals..

Reply:
yeah this makes sense. I will address it in another PR. I have made an issue about this: https://github.com/arfc/d3ploy/issues/261